### PR TITLE
Task/docker

### PR DESCRIPTION
**Pull request by gbudau**
Wordpress installed in a Docker container

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
```
Comment (Damrod) [inline docker-entrypoint.sh:2]: I like to do `set -eux -o pipefail` to my scripts, but I don't really mind.

**Review by Damrod — APPROVED** [review]
We have wordpress!
+100!!!!

File: docker-entrypoint.sh (hunk 1 above)
Comment (gbudau) [reply docker-entrypoint.sh:3]: Looks useful, done.

**Review by vdedios — APPROVED** [review]
❤️

**Comment by gbudau** [comment]
Thank you! ❤️

**Pull request merged**